src/main/resources/db/migration/V1_16__alter_innmelderType_not_null.sql

```diff
@@ -0,0 +1,11 @@
+-- This update to 1M+ rows was run manually in smaller chunks to not run out of memory
+-- UPDATE MOTEBEHOV
+-- SET innmelder_type =
+--     CASE
+--         WHEN opprettet_av = aktoer_id OR opprettet_av_fnr = sm_fnr
+--             THEN 'ARBEIDSTAKER'
+--         ELSE 'ARBEIDSGIVER'
+--         END;
+
+ALTER TABLE MOTEBEHOV
+    ALTER COLUMN innmelder_type SET NOT NULL;
```

Comment:
Bruk av NOT NULL her vil vel ikke virke. da man fra før har 1M+ rader som ikke har verdi. Man kan angi en default verdi, man da vil man ikke ha en fin måte å sjekke at jobben er ferdig med og oppdatere alle eksisterende rader